AppSettings/README.md

```diff
@@ -18,39 +18,41 @@ This will prevent code changes of how you load settings from impacting the rest
 
 Your settings class can look as simple as this:
 
-	class MySettings : SingletonSettings<MySettings>
-	{
-		[AppSetting]
-		public int MyIntValue { get; set; }
+```csharp
+class MySettings : SingletonSettings<MySettings>
+{
+    [AppSetting]
```

Comment:
Just wondering if we should just use the new .net core builder pattern instead of mash app stuff?

Reply:
We're not using the Mash.AppSettings anywhere. I just touched up that readme.md file while I was here.